enhancements/ocp-coreos-layering/ocp-coreos-layering.md

```diff
@@ -0,0 +1,725 @@
+---
+title: ocp-coreos-layering
+authors:
+  - "@yuqi-zhang"
+  - "@jkyros"
+  - "@mkenigs"
+  - "@kikisdeliveryservice"
+  - "@cheesesashimi"
+  - "@cgwalters"
+  - "@darkmuggle (emeritus)"
+  - "@sinnykumari"
+reviewers:
+  - "@mrunalp"
+approvers:
+  - "@mrunalp"
+api-approvers:
+  - "None"
+creation-date: 2021-10-19
+last-updated: 2022-06-09
+tracking-link:
+  - https://issues.redhat.com/browse/GRPA-4059
+  - https://issues.redhat.com/browse/MCO-165
+---
+
+# OpenShift Layered CoreOS
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Change RHEL CoreOS as shipped in OpenShift to be a "base image" that can be used as in layered container builds and then booted.  This will allow custom 3rd party agents delivered via RPMs installed in a container build.  The MCO will roll out and monitor these custom builds the same way it does for the "pristine" CoreOS image today.
```

Comment:
nit: used in layered..

Reply:
The MCO will roll out and monitor these custom containers..
(once built it is just like any other other container)